app/javascript/dashboard/components/ChatList.vue

```diff
@@ -623,6 +624,15 @@ export default {
         this.showAlert(this.$t('BULK_ACTION.ASSIGN_FAILED'));
       }
     },
+    async onAssignTeam(team, conversationId = null) {
+      const teamId = team ? team.id : 0;
+      await this.$store.dispatch('setCurrentChatTeam', team);
```

Comment:
This introduces a bug. If the user is right-clicking on a conversation that is not selected and assigning a team, it would assign the team to the current active conversation as well.

Reply:
@pranavrajs picked this from our contact panel component, thought this was added for an edge case. Should I remove this?

Reply:
@pranavrajs Removed this